### PR TITLE
Guile: add `devel` block for v2.1.2

### PR DESCRIPTION
**Pull request by MerelyAPseudonym**
For information about this release, see <https://lists.gnu.org/archive/html/guile-user/2016-02/msg00022.html>.

### CODE REVIEW

File: Library/Formula/guile.rb [hunk 1]
```diff
@@ -13,6 +13,14 @@ class Guile < Formula
     sha256 "51f5f379e25fab5cf8fb7cede02841aa716c0e90356705be2abc6a18c6af5371" => :mountain_lion
   end
 
+  devel do
+    url "http://git.savannah.gnu.org/r/guile.git", :tag => "v2.1.2",
+                                                   :revision => "f9490cdb3fd9c99de6976077cb87ef77c591967a"
```
Comment (mikemcquaid) [inline Library/Formula/guile.rb]: Looks wrong: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/41264/version=yosemite/testReport/junit/brew-test-bot/yosemite/fetch___devel_guile/

Comment (MerelyAPseudonym) [inline Library/Formula/guile.rb]: my bad. I used `git rev-parse v2.1.2` and forgot to check it.

File: Library/Formula/guile.rb [hunk 2]
```diff
@@ -13,6 +13,14 @@ class Guile < Formula
     sha256 "51f5f379e25fab5cf8fb7cede02841aa716c0e90356705be2abc6a18c6af5371" => :mountain_lion
   end
 
+  devel do
+    url "http://git.savannah.gnu.org/r/guile.git", :tag => "v2.1.2",
```
Comment (DomT4) [inline Library/Formula/guile.rb]: Please break the `tag` and `revision` onto their own lines, Thanks!

Comment (MerelyAPseudonym) [inline Library/Formula/guile.rb]: Sure thing! Is that something that `brew style`/ Rubocop should know to look for?

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by MerelyAPseudonym** [comment]
@DomT4 Regarding TravisCI timing out: IIRC building Guile 2.1.2 took about eighty minutes on my machine 😒

**Comment by dunn** [comment]
Merged in f79f689a623959453018c318a505481c30f24c59, thank you!

**Comment by DomT4** [comment]
Apologies for missing this @MerelyAPseudonym; It was on my to-merge list a few nights back but I restarted Chrome for a version update and it wiped my tabs.

**Pull request closed without merging**